src/common_format.rs

```diff
@@ -174,16 +173,16 @@ where
     let file = "calendar.txt";
```

Comment:
Minor: The only reason of creating this variable was to log. You can directly put the string literal in the get_file call if you want to win a line.

Reply:
but won't the log line be clustered a bit if I log the path?
`"skipping /srv/data/gtfs/calendar.txt` instead of just 
`"skipping calendar.txt` 

it's up to you, I don't really care